deaf-and-hard-of-hearing/deaf-and-hard-of-hearing-individuals/tips-when-attending-or-speaking-at-a-conference.md

```diff
@@ -0,0 +1,22 @@
+# Tips when attending or speaking at a conference that doesn't provide proper accommodations
+
+Conference organizers don't always fully understand the needs of deaf and hard of hearing attendees or speakers. As a result, conferences tend to have little to no appropriate accommodations, creating a vicious cycle of fewer deaf and hard of hearing individuals attending and speaking at events.
+
+But to change that, you need to — not only show up — but also be transparent about your needs. As you will see, the lack of accommodations isn't ill-intended. It's just due to a lack of knowledge and, other times, a lack of funds to provide required accommodations. If we don't educate them, who will? To bring change, we all need to embrace **self-advocacy** and show up and speak up. 
+
+## Tips for Attendees
+
+1. **Ask for accommodations early**: The sooner organizers know your needs, the more time they have to prepare. Organizing a conference gets increasingly stressful the closer you are to the date, so provide ample time for them to do the research. 
+2. **Share what works for you with event organizers**: If you have been to another conference where something worked for you (captions on the phone, etc.), use it as a learning opportunity for organizers.
+
+## Tips for speakers
+
+1. **Be transparent and vocal about your needs**: Publicly talk about accommodations on social media or email by replying to all to help initiate conversations. But be mindful of calling out versus calling in.
```

Comment:
I don't understand the last sentence here

Reply:
I think what @AmyJuneH is referring to here is that, instead of just emailing one person and hoping they will help, ask for it more publicly to create some social pressure to comply. I assume that is based on experience.

Reply:
I think we should reword "calling out versus calling in" to be more clear, then this PR LGTM.  I've tried a few times in my scratch pad but it's not coming across well because I don't fully understand what we're saying.  The best I have is:

Be nice but clear in your tone; seek to collaborate on solutions

Reply:
Maybe "Confrontation versus conversation" ?